action.yml

```diff
@@ -1,21 +1,22 @@
 # Meta data for the action.
 
-name: 'Review App Deployment Status'
+name: 'Heroku Review App Deployment Status'
 author: 'niteo.co'
 description: 'A Github Action to test the deployment status of a Heroku Review App.'
+branding:
+  icon: 'git-pull-request'  
+  color: 'orange'
+
 runs:
   using: 'docker'
   image: 'Dockerfile'
+
 inputs:
   interval:
-    description: (optional) Interval to check the status.
+    description: Interval to check the status.
     required: false
     default: 10 # 10 seconds
   accepted_responses:
-    description: (optional) Status(es) which can be accepted. Separated by comma.
+    description: Status(es) which can be accepted. Separated by comma.
```

Comment:
`Status(es) which can be accepted` does not tell me a lot.

Maybe something along the lines of: `The Review App needs to respond with one of these HTTP status codes in order for the HTTP check to pass.

Reply:
This is not visible to users. Just a input description. We have already provided this info in readme.